### PR TITLE
Added more info links to ProgramPage cms

### LINKED ISSUE #1983: Allow Program admins to add additional links in the More Info box
DEDP has requested the ability to add additional links in the "More Info" box on the program page. (see below). Currently in the CMS, there is only the ability to have one link there, the "PROGRAM HOME PAGE URL". We should just make this a repeatable element, and allow them to add as many links as they want.

image

image

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Fix #1983
#### What's this PR do?

Added `InfoLinks` to `ProgramPage` for user to add any addition links in the 'More Info' section.

screen shot 2016-12-05 at 5 33 40 pm

### CODE REVIEW

**Comment by amir-qayyum-khan** [comment]
please add some space between links or may be we can use bullets here?
@roberthouse54 


screen shot 2016-12-06 at 4 21 48 pm

File: cms/templates/cms/program_page.html [hunk 1]
```diff
@@ -90,7 +90,7 @@ <h3 class="title">
             <!-- CourseListWithPopover component injected here -->
           </div>
         </div>
-        {% if page.program_home_page_url or page.program_contact_email %}
+        {% if page.program_home_page_url or page.program_contact_email or page.info_links.count > 0 %}
```
Comment (amir-qayyum-khan) [inline cms/templates/cms/program_page.html]: why not just ```or page.info_links```. you are doing count check again on line 110.

**Comment by roberthouse54** [comment]
Please just increase the margin between the links. No need to use a list.

File: cms/templates/cms/program_page.html (hunk 1 above)
Comment (annagav) [inline cms/templates/cms/program_page.html]: I changed it to `page.info_links.exists`.

**Comment by annagav** [comment]
@amir-qayyum-khan I addressed your comments

**Comment by amir-qayyum-khan** [comment]
please merge once test pass 👍

**Pull request merged**